app/lib/efile/nj/nj_property_tax_eligibility.rb

```diff
@@ -0,0 +1,38 @@
+module Efile
+  module Nj
+    module NjPropertyTaxEligibility
+      INELIGIBLE = :ineligible
+      INELIGIBLE_FOR_DEDUCTION = :ineligible_for_deduction
+      NOT_INELIGIBLE = :not_ineligible
+
+      class << self
+        def determine_eligibility(intake)
+          state_wages = Efile::Nj::NjStateWages.calculate_state_wages(intake)
+
+          wage_minimum = if intake.filing_status == :married_filing_separately || intake.filing_status == :single
```

Comment:
[SAND] Prefer `intake.filing_status_mfs?` and `filing_status_single?` helper methods rather than checking directly.

Reply:
good catch - done